### PR TITLE
Workflow Automated Test - LRQA-68861 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -441,12 +441,60 @@ definition {
...
+		Type(
+			locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE",
+			value1 = " ");
+
+		Type(
+			locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE_TIME",
+			value1 = "05:30 PM");
 
-		// TODO LRQA-68861 DueDateIsRequired pending implementation
+		Workflow.confirmWorkflowAction();
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:495]: You can create and use the same macro you created at that [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/835/commits/d299c9d40897bf65014e97b95381b3a57e33cbec)

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by PauloVAS** [comment]
@lucasperj, fixed!

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**